supabase/seed.sql

```diff
@@ -4,6 +4,8 @@ insert into auth.users (id, email) values
   -- Root account which provisions other accounts.
   -- It must exist for the agent to function.
   ('ffffffff-ffff-ffff-ffff-ffffffffffff', 'support@estuary.dev'),
+  -- Account that owns the ops.$DATAPLANE collections for stats and logs.
```

Comment:
i think this should still be support@esuary.dev

Reply:
Update this (among other things) in `seed.sql`. We'll just create the one ops tenant with the specific name we want, which seems to work fine.